### ISSUE TITLE
How to use the "ext:pluploadfe" with "ext:form"?

### ISSUE THREAD

**Issue by typo3ua** [fnagel/pluploadfe#16]
Is it possible to use `ext:pluploadfe` with `ext:form` if possible then that how to do it?

**Comment by fnagel** [comment]
Not yet, but that is a nice idea!

Should be possible using a custom form element (for rendering the plupload widget and choosing a config record) and a custom finisher (for collecting the uploaded files within the session and adding them to the email template).

**Comment by typo3ua** [comment]
Yes, about finisher you are right, exactly need custom finisher because of default finisher not able save in database more than one record.

...and a custom form element needs too

**Comment by stephangrass** [comment]
ext:pluploadfe for ext:form, wow, that would be really great. Since there's ext:pluploadfe for Powermail, could you please whip up something similar for ext:form? ;-)

**Comment by fnagel** [comment]
@stephangrass Sorry, but I'm still working on TYPO3 v12 releases. Sponsoring or a PR is always possible!
